app/configurator/configurator-state.spec.tsx

```diff
@@ -1,5 +1,8 @@
 import { Client } from "@urql/core";
-import { applyDimensionToFilters } from ".";
+import {
+  applyNonTableDimensionToFilters,
+  applyTableDimensionToFilters,
+} from ".";
```

Comment:
Nit: shouldn't this be `from "./configurator-state"?

Reply:
Right, auto-imports 😅